css-three-column-layout/styles.css

```diff
@@ -0,0 +1,68 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  color: white;
+  background-color: rgb(49, 49, 233);
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+}
+
+.container {
+  max-width: 2000px;
+ margin: 0 auto;
+}
+.row {
+  display: flex;
+}
+.column-third {
+  width: calc(100% / 3);
+}
+ {
+   margin-left: 150px;
+}
+h1 {
+  margin-bottom: 100px;
+  font-size: 70px;
+}
+.dot {
+  margin-left: 300px;
```

Comment:
instead of using all this margin to control the position of the elements, I would recommend using one of the `justify-content` properties on the row class to let Flexbox control the 3 column positions